cmd/kritis/kubectl/plugins/resolve/README.md

```diff
@@ -3,46 +3,57 @@
 resolve-tags replaces tagged images in your Kubernetes yamls with their corresponding digests, and prints the new manifest to STDOUT.
 It can be run as a binary or installed as a kubectl plugin.
 
-## kubectl plugin
+## Installation
 
-To install as a kubectl plugin, run:
+## macOS
 
+```shell
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-darwin-amd64.tar && \
+RESOLVE_TAGS_DIR=$HOME/.kube/plugins/resolve && \
+mkdir -p $RESOLVE_TAGS_DIR && tar -C $RESOLVE_TAGS_DIR -xf resolve-tags-darwin-amd64.tar && \
+mv $RESOLVE_TAGS_DIR/resolve-tags-darwin-amd64 $RESOLVE_TAGS_DIR/resolve-tags && \
+sudo cp $RESOLVE_TAGS_DIR/resolve-tags /usr/local/bin/
 ```
-make install-plugin
+
+## Linux
+
+```shell
+
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-linux-amd64.tar curl  && \
```

Comment:
test this command-line - there is an extra "curl" at the end of it.

Reply:
Ahh somehow I messed with my github commits and this is an older version, sorry about that!